docs/api.md

```diff
@@ -589,3 +589,252 @@ state.
 
 Please note only the existence of the annotation is important to treat the BMH
 as detached and the value of the annotation is always ignored.
+
+## HostFirmwareSettings
+
+A **HostFirmwareSettings** resource is used to manage BIOS settings for a host,
+there is a one-to-one mapping with **BareMetalHosts**.  A
+**HostFirmwareSettings** resource is created when BIOS settings are read from
+Ironic as the host moves to the Ready state.  These settings are the complete
```

Comment:
I think the state has been renamed to `Available`

Reply:
Changed to `Available`